e2e_testing/torchscript/xfail_sets.py

```diff
@@ -85,15 +85,11 @@
     "ElementwiseReciprocalModule_basic",
     "TypePromotionAlphaWiderModule_basic",
     "Conv2dWithPaddingDilationStrideStaticModule_basic",
-    "BatchNorm1DModule_basic",
-    "BatchNorm2DModule_basic",
-    "BatchNorm3DModule_basic",
     "FlattenStaticModule_basic",
     "FlattenRank0Module_basic",
     "ElementwiseFlattenBroadcastModule_basic",
     "SquareModule_basic",
     "MaxPool2dStaticModule_basic",
-    "ResNet18StaticModule_basic",
```

Comment:
Hi @Shukla-Gaurav, I'm not sure you realized that this list is a list of *passing* tests, so this line *removes* a passing test replacing it with one that's broken! It's not ok to unilaterally turn off passing tests! Please revert this PR!

cc @anupgangwar @sjarus

Reply:
@silvasean I understand that it was a mistake to push the changes without a proper discussion with TOSA team, really sorry for that! 

either I can revert this PR right away to make the resnet functional,
or I can wait till we fix it through assert based on the discussion at https://github.com/llvm/torch-mlir/issues/618 
What do you think @silvasean ?

Reply:
We should revert it.

Reply:
Sure.